### PR TITLE
fwup 0.4.2 (new formula)

### PR DESCRIPTION
**Pull request by keyvanfatehi**
I've added a formula for `fwup`, a configurable embedded Linux firmware
update creator and runner.

Relevant links:

https://github.com/fhunleth/fwup
http://nerves-project.org/

### CODE REVIEW

File: Library/Formula/fwup.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/fwup", "-g"
+    File.exist?("fwup-key.priv") && File.exist?("fwup-key.pub")
```
Comment (mikemcquaid) [inline Library/Formula/fwup.rb]: Make these two separate `assert` so it's more obvious if one fails.

**Comment by mikemcquaid** [comment]
This repo is too nice for us for now but we might reconsider in future. Sorry! http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37373/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_fwup___strict___online/

**Comment by keyvanfatehi** [comment]
@mikemcquaid I added the asserts in my branch but if I understand correctly, I shouldn't create another PR until these conditions are met upstream? <10 forks, <10 watchers and <20 stars

Or was that an automated statement from brew bot that we don't really take too seriously?

**Comment by mikemcquaid** [comment]
> @mikemcquaid I added the asserts in my branch but if I understand correctly, I shouldn't create another PR until these conditions are met upstream? >=10 forks, =>10 watchers and =>20 stars

Yep, sorry. You may want to [create your own tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) instead.

**Pull request closed without merging**